src/routes/postSkipSegments.js

```diff
@@ -127,7 +127,7 @@ function proxySubmission(req) {
 
 module.exports = async function postSkipSegments(req, res) {
     if (config.proxySubmission) {
-        proxySubmission(req);
+        return proxySubmission(req);
```

Comment:
proxySubmission doesn't return a value though, it's intended to continue as normal even of the proxy failed.

Reply:
I see, I guess we do want the submissions to go the local database too, I didn't think about that. Yea, this can be reversed then.